modules/cloud4/modules/graph-development/pages/load-data/load-from-local.adoc

```diff
@@ -54,41 +54,42 @@ The header name is ignored during data loading.
 
 == 3) Configure Map
 
-.If you are loading data into a brand new graph, you will be prompted to let our engine generate a schema and mapping for you. Or you can start from scratch. For more details of schema design please refer to xref:cloudBeta:schema-designer:index.adoc[Schema Designer].
+.If you are loading data into a brand new graph, you will be prompted to let our engine generate a schema and mapping for you. Or you can start from scratch. For more details of schema design please refer to xref:cloud4:graph-development:design-schema/index.adoc[Design Schema].
```

Comment:
@samzhou2 Why are you using this dot syntax after a heading? What do you think it is doing?